src/procedures/proc_fulltext_query_edge.c

```
@@ -0,0 +1,218 @@
+
+	//--------------------------------------------------------------------------
+	// pull from iterator
+	//--------------------------------------------------------------------------
+
+	// try to get a result out of the iterator
+	// NULL is returned if iterator id depleted
+	size_t len = 0;
+	const EdgeIndexKey *edge_key = (EdgeIndexKey *)
+		RediSearch_ResultsIteratorNext(pdata->iter, Index_RSIndex(pdata->idx),
+				&len);
+
```

cppcheck (2.10-2)

[error] Memory leak

(memleak)